cocos/2d/CCCamera.cpp

```diff
@@ -238,9 +238,27 @@ bool Camera::initOrthographic(float zoomX, float zoomY, float nearPlane, float f
     return true;
 }
 
-void Camera::unproject(const Size& viewport, Vec3* src, Vec3* dst) const
+void Camera::project(const Size& viewport, const Vec3* src, Vec2* dst) const
 {
-    assert(dst);
+    GP_ASSERT(src);
```

Comment:
Also can we use CCASSERT, and provide correct warning messages, GP_ASSERT just a CCASSERT with out message, why are we using that?